pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -731,28 +729,33 @@ func (m *kubeGenericRuntimeManager) computePodActions(pod *v1.Pod, podStatus *ku
 					containerStatus.State != kubecontainer.ContainerStateRunning ||
 					containerStatus.ID.String() != apiContainerStatus.ContainerID ||
 					len(diff.ObjectDiff(container.Resources.Requests, container.ResourcesAllocated)) != 0 ||
-					len(diff.ObjectDiff(apiContainerStatus.Resources.Limits, container.Resources.Limits)) == 0 {
+					len(diff.ObjectDiff(apiContainerStatus.Resources, container.Resources)) == 0 {
+					continue
+				}
+				// If runtime status resources is available from CRI or previous update, compare with it.
+				if len(diff.ObjectDiff(containerStatus.Resources, container.Resources)) == 0 {
```

Comment:
is this a dup check to line 732 ?

Reply:
No. This checks runtime container status vs earlier v1 ContainerStatus from pod.Status